src/app/app.component.ts

```diff
@@ -134,17 +137,11 @@ export class AppComponent extends LogoutBasePageComponent implements OnInit {
   }
 
   configureStatusBar = async (): Promise<void> => {
-    if (Capacitor.isPluginAvailable('StatusBar')) {
```

Comment:
Does the removal of this check hinder browser development?

Reply:
it does not. serves and can go through test fine